sdk/typescript/src/types/checkpoints.ts

```diff
@@ -60,6 +63,7 @@ export const Checkpoint = object({
   epochRollingGasCostSummary: GasCostSummary,
   timestampMs: number(),
   endOfEpochData: optional(EndOfEpochData),
+  validatorSignature: ValidatorSignature,
```

Comment:
```suggestion
  // TODO(jian): remove optional after 0.30.0 is released
  validatorSignature: optional(ValidatorSignature),
```
nit: for backward compatibility